starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,78 @@
 // Names and Input
-
+let hacker1 = "Victoria"
+let hacker2 = prompt("What's the navigator name?");
+console.log("The driver's name is " + hacker1)
+console.log("The navigator name is " + hacker2)
 
 //Conditionals
+function longerName (hacker1, hacker2){
+  if (hacker1.length > hacker2.length) {
+    console.log(`The Driver has the longest name, it has ${hacker1.length} characters`)
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`Yo, navigator got the longest name, it has ${hacker2.length} characters`)
+  } else {
+    console.log(`wow, you both got equally long names, ${hacker1.length} characters!!`)
+  }
+  }
+
+  longerName(hacker1,hacker2);
+
+var capitalizedName = "";
+var reverseName = "";
+for (i = 0; i < hacker1.length; i++ ){
+  capitalizedName += " "+ hacker1[i].toUpperCase();
+}
+console.log(capitalizedName);
+
+for (a = hacker2.length; a >= 0; a--){
```

Comment:
Una pequeña corrección: si notan, hay un espacio en el início del reversedName. Eso pasa porque el índice de los caracteres en la cadena empieza en 0, pero el conteo del length empieza en 1. Para empezar en la última letra de la cadena, el valor inicial de `i` deveria de ser hacker2.length-1